src/Uno.UI/Helpers/WinUI/ResourceAccessor.cs

```diff
@@ -22,6 +22,14 @@ internal class ResourceAccessor
 		public const string MUXCONTROLS_PACKAGE_NAME = "Microsoft.UI.Xaml.3.0";
 		private const string c_resourceLoc = "Microsoft.UI.Xaml/Resources";
 
+		public const string SR_DefaultItemString = "DefaultItemString";
```

Comment:
@roblo Commented about this file in the `RatingControl` PR. I like the suggestion of a separate partial.

Also would it make sense to import all the strings in one hit? It'd probably make it easier to track upstream changes, versus bringing them in piecemeal.

Reply:
I would be happy to import all the keys at once and put them into their own partial class in a later PR. I will also go through the existing controls and make sure they don't use string literals (NumberBox).

Reply:
I have added them all too in TabView PR I think :-D . Maybe @robloo do a separate PR with just this change (moving the resources) and then we can just rebase everything on that, otherwise we will have conflicts several times :-D .

Reply:
@MartinZikmund Good point -- it will quickly get messy as at least 3 PRs are modifying this code. I'll do a PR tomorrow to import all SR keys and replace string literals in existing MUX controls. Then hopefully that can get merged into master relatively quickly and we can all rebase.

Reply:
Done in #4057